src/aotcompile.cpp

```diff
@@ -1690,6 +1702,7 @@ void jl_dump_native_impl(void *native_code,
         JL_TIMING(NATIVE_AOT, NATIVE_Setup);
         dataM.setTargetTriple(TheTriple.str());
         dataM.setDataLayout(DL);
+        dataM.setPICLevel(PICLevel::BigPIC);
```

Comment:
Why are we now requiring BigPIC?

Reply:
https://github.com/JuliaLang/julia/pull/53070#issuecomment-2002972428 It's a weird SSP thing where they are getting it from the Module instead of the TM (IMO it's an issue with their API but oh well) and we use BigPIC because that's what the image gets linked as in the end anyway (-fPIC)